capi/gen_header.sh

```diff
@@ -44,10 +47,14 @@ cp "$CAPI_DIR/include/hyper.h" "$header_file_backup"
 
 #cargo metadata --no-default-features --features ffi --format-version 1 > "$WORK_DIR/metadata.json"
 
-cd $WORK_DIR
+cd "${WORK_DIR}" || exit 2
 
 # Expand just the ffi module
-cargo rustc -- -Z unstable-options --pretty=expanded > expanded.rs 2>/dev/null
+if ! output=$(cargo rustc -- -Z unstable-options --pretty=expanded 2>&1 > expanded.rs); then
+    # As of April 2021 the script above prints a lot of warnings/errors, and
+    # exits with a nonzero return code, but hyper.h still gets generated.
+    echo "$output"
```

Comment:
I've noticed that all PRs now get Github Actions Annotations with the warnings from this script, and I suspect it's automatically doing it from this output. Do you think it'd be horrible if we silence this if `--verify` was passed?

Reply:
No that's alright